jcl/src/openj9.jvm/share/classes/com/ibm/jvm/Dump.java

```diff
@@ -671,9 +684,13 @@ public static void resetDumpOptions() throws DumpConfigurationUnavailableExcepti
     	}
     }
 
-	private static native void setDumpOptionsImpl(String options) throws InvalidDumpOptionException, DumpConfigurationUnavailableException;
+	private static native int JavaDumpImpl();
+	private static native int HeapDumpImpl();
+	private static native int SnapDumpImpl();
+	private static native int SystemDumpImpl();
+	private static native void setDumpOptionsImpl(String options) throws InvalidDumpOptionExceptionBase, DumpConfigurationUnavailableExceptionBase;
 	private static native String queryDumpOptionsImpl();
 	private static native void resetDumpOptionsImpl() throws DumpConfigurationUnavailableException;
```

Comment:
This can't throw `DumpConfigurationUnavailableException`; instead it may throw `DumpConfigurationUnavailableExceptionBase`.

Reply:
Added a try/catch/wrap & rethrow, as elsewhere.